packages/emotion-theming/src/use-theme.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+import * as React from 'react'
```

Comment:
Doing 👇 will mean that emotion-theming won't break for people who aren't on `>=16.8.0`

```suggestion
import React from 'react'
```

Reply:
@mitchellhamilton `import * as React from 'react'` was always supported, this ain't nothing to do with react@16.8. And also - the wildcard is somewhat more correct than the default export. Or maybe you were referencing something I'm not aware of.

Reply:
I was thinking that it would fail for users who aren't on `>=16.8.0` because rollup would change `import * as React from 'react`' to `import { useContext } from 'react'` and that export wouldn't exist so it would fail. Though I guess since React is CJS, it wouldn't have failed? I'd still rather leave it as `import React from 'react'` though just in case someone uses an ESM bundle of React and aliased it or rollup users with rollup-plugin-commonjs because of the `namedExports` option or something like that.

Reply:
Interesting edge cases. I guess technically they are possible so yeah - let's keep the default for now.